documentation/Database-Schema-Permission.md

```diff
@@ -0,0 +1,34 @@
+# Database Schema Permission
+
+![Database Schema](images/database-schema-permission.png)
```

Comment:
You've actually misspelled the .png file (I can't comment on it directly). I only noticed cos it wasn't rendering in the page preview -- 😬

Reply:
Also -- image is cut off, can only see 5 tables at the top.

Reply:
Oh, that's a strange bug on DBeaver. I have fixed now.